pam/integration-tests/vhs-helpers_test.go

```diff
@@ -238,22 +254,38 @@ func (td *tapeData) AddClientOptions(t *testing.T, opts clientOptions) {
 	}
 }
 
-func (td tapeData) RunVhs(t *testing.T, testType vhsTestType, outDir string, cliEnv []string) {
+func (td *tapeData) RunVHS(t *testing.T, testType vhsTestType, cliEnv []string) {
 	t.Helper()
 
 	cmd := exec.Command("env", "vhs")
+
+	var outBuf testutils.SyncBuffer
+	// Write stdout/stderr both to our stdout/stderr and to the buffer
+	cmd.Stdout = io.MultiWriter(t.Output(), &outBuf)
+	cmd.Stderr = io.MultiWriter(t.Output(), &outBuf)
+
 	cmd.Env = append(testutils.AppendCovEnv(cmd.Env), cliEnv...)
-	cmd.Dir = outDir
+	cmd.Dir = td.OutputDir
 
 	cmd.Env = append(cmd.Env,
 		// If vhs is installed with "go install", we need to add GOPATH to PATH.
-		prependBinToPath(t),
+		pathEnvWithGoBin(t),
 		// vhs uses rod, which downloads chromium to $HOME/.cache/rod,
 		// so $HOME needs to be set to avoid that it downloads it every time.
 		// TODO: Set XDG_CACHE_HOME instead once https://github.com/go-rod/rod/pull/1213 was merged
 		"HOME="+os.Getenv("HOME"),
+		// Avoid VHS printing "Host your GIF on vhs.charm.sh: vhs publish <file>.gif"
+		"VHS_PUBLISH=false",
 	)
 
+	// Force VHS to use color because it makes it a lot easier to find the error
+	// message in the VHS output. The only case where we don't want color is when
+	// building a Debian package, because when viewing the logs of a launchpad build
+	// in the browser, ANSI colors are not rendered.
+	if os.Getenv("DEB_BUILD_ARCH") == "" {
```

Comment:
Didn't we say to do:


```suggestion
	if !testutils.IsCI() && os.Getenv("DEB_BUILD_ARCH") == "" {
```